superset-frontend/src/explore/components/controls/DatasourceControl/index.jsx

```diff
@@ -190,7 +195,7 @@ class DatasourceControl extends React.PureComponent {
     return (
       <Styles data-test="datasource-control" className="DatasourceControl">
         <div className="data-container">
-          <Icon name="dataset-physical" className="dataset-svg" />
+          <Icons.DatasetPhysical className="dataset-svg" />
```

Comment:
conditional render `DatasetPhysical` or `DatasetVirtual` from `props.datasource.is_sqllab_view`

Reply:
I disagree, I think having a consistent icon here adds clarity. To me the icon is doing the job of communicating "this is the dataset control" rather than "this is the type of this dataset". But that decision should probably be left to design folks.

Reply:
I'm fine either way, but I liked @zhaoyongjie suggestion. The icons are similar in a way that the user knows that they represent a dataset but with different icons, we have additional information (virtual or physical).

<img width="38" alt="Screen Shot 2021-07-08 at 5 04 54 PM" src="https://user-images.githubusercontent.com/70410625/124983675-a3f77680-e00e-11eb-9a74-a9ea67f85523.png">

Reply:
One question that we can ask is: In the context of Explore, is important for the user to know if he's working on a physical or virtual dataset? Is there any difference in terms of functionality?

Reply:
The original design for this control was with this specific icon.

Reply:
@michael-s-molina it shouldn't matter, everything the user does on a physical dataset in Explore they should be able to in a virtual, and vice-versa.

Reply:
I vote for consistency in design